app/views/govuk_publishing_components/components/docs/checkbox.yml

```diff
@@ -0,0 +1,39 @@
+name: Checkbox
+description: Let users select one or more options by using the checkboxes component.
+govuk_frontend_components:
+  - checkboxes
+accessibility_criteria: |
+  The component must:
+
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - be usable with touch
+  - indicate when they have focus
+  - be recognisable as form textarea elements
```

Comment:
Looks like you may have copied and pasted some of these from somewhere else, some of them don't seem relevant.